### PR TITLE
Fixed the console  error in Imageview.dart

### LINKED ISSUE #71: Selecting an image from the gallery gives an error on the console
The error is described below

```
======== Exception caught by widgets library =======================================================
The following assertion was thrown while applying parent data.:
Incorrect use of ParentDataWidget.
```

Here is the full error
```
======== Exception caught by widgets library =======================================================
The following assertion was thrown while applying parent data.:
Incorrect use of ParentDataWidget.

The ParentDataWidget Expanded(flex: 1) wants to apply ParentData of type FlexParentData to a RenderObject, which has been set up to accept ParentData of incompatible type BoxParentData.

Usually, this means that the Expanded widget has the wrong ancestor RenderObjectWidget. Typically, Expanded widgets are placed directly inside Flex widgets.
The offending Expanded is currently placed inside a Padding widget.

The ownership chain for the RenderObject that received the incompatible parent data was:
  ConstrainedBox ← Container ← Expanded ← MediaQuery ← Padding ← SafeArea ← Column ← ColoredBox ← Container ← MediaQuery ← ⋯
When the exception was thrown, this was the stack: 
#0      RenderObjectElement._updateParentData.<anonymous closure> (package:flutter/src/widgets/framework.dart:5770:11)
#1      RenderObjectElement._updateParentData (package:flutter/src/widgets/framework.dart:5786:6)
#2      RenderObjectElement.attachRenderObject (package:flutter/src/widgets/framework.dart:5808:7)
#3      RenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5501:5)
#4      SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:6117:11)
...
====================================================================================================
```

I'd like to work on this issue as a part of MWoC.

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Closes #71 

Deleted the SafeArea and changed the flex for both the Expanded widgets which now are in a single column. This fixes the console error. Please let me know if you want any further changes in the PR. Thank you.

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
image
Overflow error is occurring due to removing the safe area widget please solve this too

**Comment by Saransh-cpp** [comment]
Please don't merge right now, I am making some more changes.

**Comment by Saransh-cpp** [comment]
Fixed everything now, I missed some Animated pushes and another console error and a following renderflex overflow from Filterview page. Opening other issue would have been repetitive so I fixed them in this PR. Let me know if you want any changes. Thank you.

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**
